interview-questions.md

```diff
@@ -9,59 +9,61 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer:
+  Your answer: I honestly wasn't able to recall the lifecycle methods off the top of my head.
 
-  Researched answer:
+  Researched answer:The render() method is required and will always be called, the others are optional and will be called if you define them.
 
 
 
 4. What is JSX? What is one notable difference between HTML and JSX?
 
-  Your answer:
+  Your answer: JSX is essentually a hybrid of Javascript and HTML that is used in React.
+
+  Researched answer:JSX is a Javascript syntax extension used by ReactJS that can write Html-like Java scripts.  HTML is a hypertext language used to create items that appear on the screen, such as lists, divs, and images. JSX, on the other hand, is a JavaScript extension that allows you to create HTML in JavaScript code. The biggest difference between JSX and HTML is that nested JSX returns only one element. In other words, you still need a parent to create sibling items. It is not required for HTML.
 
-  Researched answer:
 
 
 
 5. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-  Your answer:
+  Your answer: Yarn seems to install the nessisary componets for React as well as acting as local host servers that allow users to run React applications on their local machines.
+
+  Researched answer:Yarn is the package manager for your code. You can use code to share with other developers around the world. Yarn does this quickly, safely and reliably, so you don't have to worry. Yarn allows you to use other developers' solutions to a variety of problems, making it easier to improve your software. If you encounter a problem, you can report the issue, contribute to GitHub, and once the issue is resolved, you can use Yarn to keep everything up-to-date. The code is released through so-called packages. The package contains all the shared code and the package.json file (called the manifest) that describes the package.
```

Comment:
Great!  Just be sure that you are phrasing this in your own words.  While it is a researched answer, we want to see how you interpret your research from the internet.